client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -53,11 +56,13 @@ export function getEnhancedTasks(
 					taskData = {
 						title: translate( 'Add links' ),
 						actionUrl: `/site-editor/${ siteSlug }`,
+						isCompleted: linkInBioLinksEditCompleted,
 					};
 					break;
 				case 'link_in_bio_launched':
 					taskData = {
 						title: translate( 'Launch Link in bio' ),
+						isCompleted: ! linkInBioLinksEditCompleted,
```

Comment:
This logic might get confusing to read. When we first land in the link in bio launchpad, the "launch link in bio" task hasn't actually been completed. We'd just like to disable it until the "add links" step has been completed.

How would we feel about adding some sort of dependencies array?

```
taskData = {
  title: translate( 'Launch Link in bio' ),
  isCompleted: linkInBioSiteLaunchCompleted,
  dependencies: [ linkInBioLinksEditCompleted ]
}
```

We could then read the dependencies array to help determine whether or not a particular task should be disabled.

Reply:
I think that's a good idea. That should make it easier to read and manage when tasks are enabled or disabled. We briefly discussed this yesterday. I can create a function that will validate if all the dependencies are true in order to enable/disable a specific task.